src/api/app/models/staging_workflow.rb

```diff
@@ -11,7 +11,8 @@ def without_staged_requests
   after_create :create_staging_projects
 
   def unassigned_requests
-    target_of_bs_requests.in_states(['new', 'review']) - staged_requests - ignored_requests
+    # quick and dirty, needs to be rewritten
```

Comment:
We can drop the comment and move this change to a dedicated commit? I think it is good enough for now, it is an ActiveRecord collection which is already way better than before. There are also changes from @mdeniz PR coming in, we can have another look then.